### ISSUE TITLE
Object goal navigation

### ISSUE THREAD

**Issue by leyuan-sun** [JeremyLinky/YouTube-VLN#1]
Is it possible to adapt it to ObjNav task?
How to evaluate the objnav task used your pre-trained model?

**Comment by JeremyLinky** [comment]
Sure! Hopefully, it can be adapted to object goal navigation by replacing the instructions with goals. But I think this requires more semantic knowledge or labels during both pertaining and fine-tuning. Our model could be used to test with minor modifications but might not be optimal for object goal navigation.
